pkg/kubelet/metrics/metrics.go

```diff
@@ -79,6 +79,19 @@ const (
 	// Metrics keys for RuntimeClass
 	RunPodSandboxDurationKey = "run_podsandbox_duration_seconds"
 	RunPodSandboxErrorsKey   = "run_podsandbox_errors_total"
+
+	// Metrics to keep track of objects under management
+	ManagedPodsKey                  = "managed_pods"
+	ManagedContainersKey            = "managed_containers"
+	StartedPodsTotalKey             = "started_pods_total"
+	StartedPodsErrorsTotalKey       = "started_pods_errors_total"
+	StartedContainersTotalKey       = "started_containers_total"
```

Comment:
Same as above we have containers and init containers. Do you plan on adding init containers metrics as well?

Reply:
Yes, the type of container is stored using the label `container_type` rather than a separate metric.